ggml/include/ggml.h

```diff
@@ -2507,6 +2507,9 @@ extern "C" {
     GGML_API int ggml_cpu_has_cann       (void);
     GGML_API int ggml_cpu_has_llamafile  (void);
 
+    // get the sve vector length in bytes
+    GGML_API int ggml_cpu_get_sve_cnt(void);
```

Comment:
This function is better to not be facing the public API. Will merge this for now, but consider making it private.